Lib/fontTools/varLib/__init__.py

```diff
@@ -761,6 +766,7 @@ def load_designspace(designspace):
 		masters,
 		instances,
 		ds.rules,
+		getattr(ds, "rulesProcessingLast", False),
```

Comment:
Perhaps I'm being too cautious here, but a ds object can be passed to `load_designspace()`, which could potentially be duck-typing a DesignSpaceDocument. If that's to far-fetched I'll simplify.

Reply:
well that could be said for any attribute... anyway, it doesn't matter.

Reply:
No, because it currently demands certain attrs to be there. All I wanted to do is prevent working code to break because of a change I made. It's a change in contract after all. But again, it may be overly cautious...